config/webpack/loaders/devSiteExample.js

```diff
@@ -11,6 +12,31 @@ const loader = async function loader() {
   const exampleSource = this.resourcePath;
   const parsedResourcePath = path.parse(exampleSource);
 
+  let cssFileName;
+  let exampleCssSource;
+
+  /**
+   * Use a try catch block in order to synchronously capture the file name for any attached css files
+   */
+  try {
+  // eslint-disable-next-line import/order
+    const lineReader = require('readline').createInterface({
+      input: require('fs').createReadStream(exampleSource),
+      crlfDelay: Infinity,
+    });
+
+    lineReader.on('line', (line) => {
+      if (line.includes('.scss') === true || line.includes('.css' === true)) {
+        cssFileName = (line.slice(line.lastIndexOf('/'), (line.lastIndexOf('css') + 3)));
+      }
+    });
+
+    await require('events').once(lineReader, 'close');
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    console.error(err);
+  }
+
```

Comment:
Can't we just replace this with say a regex? The contents of the file is passed into this loader as a string. We can just use that instead of calling out to the file system again.

To access content you can just add the variable to the function definition:
```js
const loader = async function loader(content) {
//Do something to read that content string and get the css import
};
```

Reply:
Simplified the code: [f086e80](https://github.com/cerner/terra-dev-site/pull/281/commits/f086e8097bc5393e9187cd5ca9efb848e1dcb773).